var/spack/repos/builtin/packages/py-gmsh-interop/package.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyGmshInterop(PythonPackage):
+    """Interoperability between Python and Gmsh"""
+
+    homepage = "https://documen.tician.de/gmsh_interop"
+    git = "https://github.com/inducer/gmsh_interop"
```

Comment:
https://spack.readthedocs.io/en/latest/build_systems/pythonpackage.html#pypi-vs-github

Reply:
I added a `pypi` member variable to the packages available from PyPI. Is there an easy way for me to validate my changes?

Reply:
You can try `spack fetch py-gmsh-interop` to make sure that it downloads properly.